make/tools.mk

```diff
@@ -26,7 +27,7 @@ bin/scratch/tools:
 	@mkdir -p $@
 
 .PHONY: tools
-tools: bin/tools/helm bin/tools/kubectl bin/tools/kind bin/tools/cosign bin/tools/release-notes bin/tools/cmrel bin/tools/ytt bin/tools/yq
+tools: bin/tools/helm bin/tools/kubectl bin/tools/kind bin/tools/cosign bin/tools/cmrel bin/tools/release-notes bin/tools/goimports bin/tools/ytt bin/tools/yq
```

Comment:
note: I rearranged cmrel to come before release-notes just to match the ordering we have in the list of VERSION variables above.